docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,26 @@
 title: Changelog
 ---
 
+## 13.2.0
+
+_Released 09/12/2023_
+
+**Features:**
+
+- Adds support for Nx users who want to run Angular Component Testing in parallel. Addresses [#27723](https://github.com/cypress-io/cypress/pull/27723).
+
+**Bugfixes:**
+
+- Edge cases where `cy.intercept()` would not properly intercept and asset response bodies would not properly be captured for test replay have been addressed. Addressed in [#27771](https://github.com/cypress-io/cypress/pull/27771).
+- Fixed an issue where `enter`, `keyup`, and `space` events where not triggering `click` events properly in some versions of Firefox. Addressed in [#27715](https://github.com/cypress-io/cypress/pull/27715).
+- Fixed a regression in `13.0.0` where tests using Basic Authorization can potentially hang indefinitely on chromium browsers. Addressed in [#27781](https://github.com/cypress-io/cypress/pull/27781).
+
+**Dependency Updates:**
+
+- Upgraded Electron from `21.0.0` to `25.8.0`, which updates bundled Chromium from `106.0.5249.51` to `114.0.5735.289`. Addresses [#27595](https://github.com/cypress-io/cypress/issues/27595).
+  Additionally, the Node version the binary has been upgraded from from `16.16.0` to `18.15.0`. This does **NOT** have an impact on the node version you are using with Cypress and is merely an internal update to the repository & shipped binary.
+  Addressed in [#27715](https://github.com/cypress-io/cypress/pull/27715). Addresses [#27595](https://github.com/cypress-io/cypress/issues/27595).
```

Comment:
The paragraph contains

`Addressed in [#27715](https://github.com/cypress-io/cypress/pull/27715).`

twice. Perhaps one of the instances can be removed?

Reply:
@AtofStryker 

Yes, sorry. Copy and paste error!

Reply:
no worries just wanted to make sure. I think this originally was intended as two entries and instead became one. We can get rid of it. Going to push an update because I need to run the linting job. should be addressed in 6fbf1dbd1